lib/server/index.js

```diff
@@ -187,6 +187,13 @@ Server.prototype = {
     if (proxies) {
       self.proxy = new httpProxy.createProxyServer({changeOrigin: true});
 
+      self.proxy.on('error', function (err, req, res) {
+        res.writeHead(500, {
+          'Content-Type': 'text/plain'
+        });
+
+        res.end('Something went wrong. And we are reporting a custom error message.');
```

Comment:
Sorry, I got confused. Logging the error itself and the stack should be fine for now. As testem shouldn't be used in production the stack can also be part of the response for easier development